keep-ui/features/workflows/builder/ui/BuilderChat/AddStepUI.tsx

```
@@ -0,0 +1,214 @@
+import { Button } from "@/components/ui";
+import { useWorkflowStore, V2Step } from "@/entities/workflows";
+import { WF_DEBUG_INFO } from "../debug-settings";
+import { DebugArgs } from "./debug-args";
+import { StepPreview } from "./StepPreview";
+import { SuggestionStatus } from "./SuggestionStatus";
+import clsx from "clsx";
+import { DebugJSON } from "@/shared/ui";
+import { useCallback } from "react";
+import { triggerTypes } from "../../lib/utils";
+
+type SuggestionStatus = "complete" | "error" | "declined";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'SuggestionStatus'. Consider to delete it or rename it.

'SuggestionStatus' is defined here:


(lint/suspicious/noRedeclare)